tasks/datasources.yml

```diff
@@ -23,6 +23,8 @@
       datasources:
       {{ grafana_datasources | to_nice_yaml }}
     backup: false
+    owner: {{ grafana_files.owner }}
```

Comment:
Grafana installation always creates user `grafana` so it doesn't make sense to customize this setting.

Similar case is with group setting. We want to force users into having secure setup, so if it works with `root` as a group, then let's use it statically.

Reply:
It could also be a good idea to add `mode` option to define file permissions.

Reply:
Ok, I was maybe wanting to do it a little more complex as required in order to let the user chose which ownership it needs (thinking on some weird packaging of grafana on some distro, but if this is not the case, you're right this somehow doesn't make sense). In order to get secure setup I won't change the mode as it may expose files in a not wanted manner. I'll update the PR with your suggestion.